rbac/internal/utils.py

```diff
@@ -91,3 +93,31 @@ def delete_bindings(bindings):
             bindings.delete()
         info["relations"] = [stringify_spicedb_relationship(relation) for relation in relations_to_remove]
     return info
+
+
+def get_or_create_ungrouped_workspace(tenant):
```

Comment:
do we have this logic already somewhere ? can it be  reused ? (in internal view retrieve_ungrouped_workspace)